.circleci/config.yml

```diff
@@ -86,8 +86,7 @@ jobs:
           command: |
             if [ -n "${npm_TOKEN+x}" ] && [ "${CIRCLE_BRANCH}" == "master" ]; then
               echo "//registry.npmjs.org/:_authToken=${npm_TOKEN}" > ~/project/.npmrc
-              npm version prerelease --preid="${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}.`git rev-parse --short HEAD`" --no-git-tag-version
-              npm publish --tag current
+              yarn lerna publish --canary --yes
```

Comment:
could we presume `preid` value used before - just to avoid potential clashes in versioning, as hash value in generated version is defined by git commit. Like
```
lerna publish --canary --preid "${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}"
```

Reply:
Sure, done 👍 

Will give us: `0.19.3-%CIRCLE_BUILD_NUM%.%CIRCLE_BRANCH%.23+002c6557b`